src/i18n/strings/nl.json

```diff
@@ -199,6 +199,7 @@
     "%(targetName)s joined the room.": "%(targetName)s is tot het gesprek toegetreden.",
     "Jump to first unread message.": "Spring naar het eerste ongelezen bericht.",
     "Labs": "Experimenteel",
+    "LaTeX math in messages": "LaTeX wiskunde in berichten",
```

Comment:
translating like this causes conflicts - please only use the output of `yarn i18n` and don't populate other json files.

Reply:
reverted. I don't fully understand how i18n works, so I'll leave this to someone else.